guides/common/modules/con_managing-content-views.adoc

```diff
@@ -1,7 +1,7 @@
 [id="Managing_Content_Views_{context}"]
 = Managing Content Views
 
-{ProjectName} uses Content Views to create customized repositories from the repositories.
+{ProjectName} uses Content Views to create customized repository sets from the imported repositories.
```

Comment:
A "repository set" already has a specific Katello definition.  "Imported" also has a specific Katello definition.

 How about "...uses Content views to create customized groupings of repository snapshots."

Reply:
Applied